Src/Json/Shared.Json/ObjectDiffPatch.cs

```diff
@@ -0,0 +1,378 @@
+﻿#region *   License     *
+/*
+    SimpleHelpers - ObjectDiffPatch   
```

Comment:
Sadly, json.net has only `JToken.DeepEquals`. The NuGet package is https://www.nuget.org/packages/SimpleHelpers.ObjectDiffPatch/. I can factor this out if you want. However, it's a MiniMod, i.e. will be under FA source control.